### DISCUSSION TITLE
Apple TV Remote done the same as this

### DISCUSSION THREAD

**Discussion by Paradox8** [PRProd/HA-Firemote#127] [Ideas]
Sorry for asking this here but I wasn't sure where else to put this forth. This Firemote card is absolutely amazing, and looks great I use it in lieu of my Shield Remote this days. Anyway, one of my other TV's has an Apple TV on it just wondering if there is any chance in the world you would be able to do the same thing as this but for Apple TV. 

The Apple TV integration is used by over 11% of active HA installations so clearly a very heavily used platform (by comparison, the ADB integration is only used by 4%).

No dramas at all if you're not interested - you've just done so amazingly well with this that I thought you would be the best person for the job.

**Comment by PRProd** [comment]
Thanks for the kind words!  I have been taking a poll, and moving on to the next things as the community votes and/or by whatever device ends up in my hands next.  

Check out discussion #41 and make your voice heard there.  😀

**Comment by PRProd** [comment]
@Paradox8 - I see that you added your vote over in #41, so I'll close this discussion for now.  Thanks for your contribution!
